### DISCUSSION TITLE
关于在 Windows 下无法切换中英文的问题

### DISCUSSION THREAD

**Discussion by LufsX** [LufsX/rime#5] [General]
Windows 用户如果想使用小狼毫提供的英文输入法的话，需要先将 https://github.com/LufsX/rime/blob/master/lufs_pinyin.schema.yaml#L32 和 https://github.com/LufsX/rime/blob/master/lufs_flypy.schema.yaml#L33 的注释取消，重新部署后即可使用右 Shift 键切换中英文输入了

如需自定义中英文切换按键，可在 https://github.com/LufsX/rime/blob/master/default.custom.yaml#L14-L19 自定义，按键行为一般都填写 `commit_code`

**Comment by kiyurei** [comment]
我想用小狼毫输入英文，请问应该怎么改？小写还好，大写感觉有点问题，也可能只是我不太习惯

**Reply by LufsX** [reply]
第一个问题的话，像上面那样改就行了

第二个应该是大写锁定不适应吧，你把 https://github.com/LufsX/rime/blob/master/default.custom.yaml#L18 删掉即可

**Reply by kiyurei** [reply]
大写的话会有个方框把他们连在一起，还有图标，我想得是一个字母一个字母打上去

![JPEGView_DHc2TxAWco](https://user-images.githubusercontent.com/101028325/196399712-2468250e-1d20-4881-9112-5ebe00a95eb6.png)

**Reply by LufsX** [reply]
哪把这边的[快捷输入](https://github.com/LufsX/rime/blob/master/lufs_flypy.schema.yaml#L195-L200)（[全拼的](https://github.com/LufsX/rime/blob/master/lufs_pinyin.schema.yaml#L181-L186)）删掉应该就可以了
